_layouts/docs.html

```diff
@@ -14,269 +14,10 @@
-                                <div id="side-toc-title">On this page:</div>
-                                    {% include toc_pure_liquid.html html=content sanitize=true class="inline_toc" id="my_toc" toc_min=my_min toc_max=my_max page_name=my_name %}
-                                {% endunless %}
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </div>
-
-    {% if page.path == "404.md" %}
-    <script type="text/javascript">
-        var path = window.location.pathname.replace("#","");
-        var ghIssueQueryString = "https://github.com/docker/docker.github.io/issues/new?" + "title=404%20at%20" + path + "&body=URL%3A%20" + path;
-        document.getElementsByClassName("nomunge")[1].onclick = function() {
-             document.getElementsByClassName("nomunge")[1].href = ghIssueQueryString;
-             return true;
-        };
-    </script>
-    {% endif %}
-    <footer class="footer">
-        {% include footer.html %}
-    </footer>
-    <link rel="stylesheet" href="/css/github.css">
-    {% if page.highlightjs == true %}
-    <script src="/js/highlight.pack.js"></script>
-    <script>
-        hljs.initHighlightingOnLoad();
-    </script>
-    {% endif %}
-    <script defer src="/js/anchorlinks.js"></script>
-    <script defer src="/js/menu.js"></script>
-    <script src="/js/jquery.js"></script>
-    <script src="/js/bootstrap.min.js"></script>
-    <!-- Always include the archive.js, but it doesn't do much unless we are an archive -->
-    <script>
-    // Default to assuming this is an archive and hiding some stuff
-    // See js/archive.js and js/docs.js for logic relating to this
-    var isArchive = true;
-    var dockerVersion = 'v{{ site.docker_ce_version }}';
-    // In archives, we need to know the page root and we get it from JEKYLL_ENV in the jekyll build command
-    var jekyllEnv = '{{ jekyll.environment }}';
-    // If unset (in non-archive branches), defaults to "development". In that case, reset it to empty
-    if (jekyllEnv === 'development') {
-        jekyllEnv = '';
-    }
-    var pageURL = jekyllEnv + '{{ page.url }}';
-    </script>
-    <script src="/js/archive.js"></script>
-    <script src="/js/stickyfill.min.js"></script>
-    <script defer src="/js/metadata.js"></script>
-    <script src="/js/glossary.js"></script>
-    <script defer src="/js/docs.js"></script>
-    <script defer src="/js/toc.js"></script>
-    <script>
-    jQuery(document).ready(function(){hookupTOCEvents();});
-    </script>
-</body>
+{% include head.html %}
+{% if page.landing == true %}
+    {% include body-landing.html %}
+{%else%}
+    {% include body.html %}
+{% endif %}
```

Comment:
Niceee 👍 


(was considering using a different `_layout` for the landing page instead of re-using the same for everything, but this looks clean)